test/k8s.js

```diff
@@ -210,3 +212,74 @@ test('all except kubernetesPodUID', function (t) {
     client.flush()
   })
 })
+
+test('Tests for ../lib/container-info ', function (t) {
```

Comment:
nit: Drop the trailing space on the test name.

Reply:
rid applied.